README.md

```diff
@@ -0,0 +1,96 @@
+# Complete integration sample for the Corbado web component
+This is a sample implementation of frontend and backend where the Corbado web component is integrated.
+
+## 1. File structure
+    .
+    ├── ...
+    ├── config                        
+    │   └── routes.yaml                 # Assigns paths to controller methods    
+    ├── src                             
+    │   ├── Controller                  
+    │   │   ├── BackendController.php   # Manages endpoints for backend
+    │   └── └── FrontendController.php  # Manages endpoints for frontend
+    ├── templates                     
+    │   ├── home.html.twig              # Home page which you only get to see if you are logged in
+    │   └── login.html.twig             # Login page which contains the Corbado web component
+    └── ...
+
+## 2. How to use
+>**Warning**
+>This sample code corresponds to the integration tutorial in our docs, please read it first in order to understand the flows and business logic!
+
+### 2.1. Prerequisites
+The only thing you need is a CNAME which points to `auth.corbado.com`. We will use `auth.your-company.com` in this tutorial. More info on what a CNAME is and why it is needed can be found in our [docs](https://docs.corbado.com/integrations/web-component#1.-define-cname).
+
+### 2.2. Corbado developer panel settings
+
+>**Warning**
+>Remember to always press "Save changes" after entering details in the Corbado developer panel!
+>
+#### 2.2.1. Configure CNAME
+
+In the developer panel under `Project settings -> Web component` enter the CNAME you previously created. 
+![image](https://user-images.githubusercontent.com/23581140/205950309-f6f622e5-94ca-4413-9384-d7a2605da75d.png)
+
+#### 2.2.2. Fill in your backend endpoints
+
+Under `Project settings -> Web component` configure the endpoints as seen in the following image, but use your own ngrok url (the paths stay the same).
+```
+<your-ngrok-url>/api/sessionToken
+<your-ngrok-url>/api/loginInfo
+<your-ngrok-url>/api/passwordVerify
+```
+![image](https://user-images.githubusercontent.com/23581140/205945743-207cd062-bb41-4b3c-af0c-cb13bf279f9c.png)
+
+#### 2.2.3. Authorize origins
+Inside the developer panel under `Project settings -> REST API` you need to enter the following origins in order to allow them to share resources.
+1. `https://CNAME`, which represents the Corbado web component
+2. `http://localhost:8000`, which is where your backend instance is running
+3. Your ngrok url which connects your local backend to the world
+
+![image](https://user-images.githubusercontent.com/23581140/205950485-6285d536-d676-4382-a23c-c3c0bbfe3de4.png)
+
+### 2.3. Setup
```

Comment:
kannst du das bitte auch noch in weitere Substeps 2.3.x unterteilen, sonst wirkt das wie so ein großer Block -> kleine Schritte sind besser